roles/kubernetes-apps/csi_driver/azuredisk/templates/azure-csi-azuredisk-node.yml.j2

```diff
@@ -150,9 +150,9 @@ spec:
             path: /var/lib/kubelet/plugins_registry/
             type: DirectoryOrCreate
           name: registration-dir
-        - hostPath:
-            path: /etc/kubernetes/
-            type: DirectoryOrCreate
+        - secret:
+            defaultMode: 420
```

Comment:
This is an interesting file mode (o=r,g=w,o=nothing), are you sure about this?

Reply:
I've updated it to [default ](https://kubernetes.io/docs/concepts/configuration/secret/#secret-files-permissions) `0644`, but it's octal, and will be converted to `420` (decimal anyway). But it does not change a lot in target pod anyway for whatever reason.
The thing I'm more concerned about is the correct source of the mount. The secret is already there and contains way less data then whole /etc/kubernetes, so it seems a better way to use it. But I think we could still mount /etc/kubernetes as it was [changing just a secret name](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes-apps/csi_driver/azuredisk/templates/azure-csi-azuredisk-node.yml.j2#L91) from cloud-config (which is wrong) to cloud_config (which is existing file in /etc/kubernetes).

Reply:
Octal format is easier to process for old-time unix folks like me and YAML unlike JSON does support octal. Thanks for updating.